lib/syntax/node/Layer.js

```diff
@@ -8,16 +8,20 @@ export const structure = {
 };
 
 export function parse() {
+
+    let tokenStart = this.tokenStart;
+    let tokenEnd = this.tokenEnd;
```

Comment:
No need for `tokenEnd`, it will be available as `this.tokenStart` (the end of consumed tokens will be a start of parser's current token) at the function return.

Reply:
Ah good call.